src/plugins/chart_expressions/expression_partition_vis/public/expression_renderers/partition_vis_renderer.tsx

```diff
@@ -62,9 +66,11 @@ export const getPartitionVisRenderer: (
       const visualizationType = extractVisualizationType(executionContext);
 
       if (containerType && visualizationType) {
-        plugins.usageCollection?.reportUiCounter(containerType, METRIC_TYPE.COUNT, [
+        const events = [
           `render_${visualizationType}_${visType}`,
-        ]);
+          canNavigateToLens ? `render_${visualizationType}_${visType}_convertable` : undefined,
+        ].filter(Boolean) as string[];
+        plugins.usageCollection?.reportUiCounter(containerType, METRIC_TYPE.COUNT, events);
```

Comment:
What is this going to measure if the events are undefined? Why don't we allow the reportCounter only if events are defined?

Reply:
@stratoula, as I see from the code, `events` will always contain `render_${visualizationType}_${visType}`. WDYT?